postgresql-dedicated-clusters-beta.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Announcing the public beta for dedicated clusters'
+description: "Today we're excited to open up dedicated clusters to all our customers and some amazin new additions to our serverless Postgres platform."
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/images/postgresql-dedicated-clusters-beta/dedicated-clusters.jpg
+  alt: Announcing the public beta for dedicated clusters
+author: Alex Francoeur
+authorEmail: alexf@xata.io
+date: 07-15-2024
+tags: ['launch-week', 'postgres', 'elephant-on-the-move']
+published: true
+slug: postgresql-dedicated-clusters-beta
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/images/postgresql-dedicated-clusters-beta/dedicated-clusters.jpg
+---
+
+Earlier this year we announced our [serverless Postgres platform](https://xata.io/blog/serverless-postgres-platform) and an early access program for [dedicated clusters](https://xata.io/blog/postgres-dedicated-clusters). Both of these offerings are core to Xata's evolution to becoming a Postgres database platform for faster development and a worry-free data layer. Serverless Postgres unlocks decades worth of community, tooling and support for our customers. While dedicated clusters provide more predictable costs, better performance and a more scalable solution for large production workloads. The two pair perfectly together, and we'll be diving into what's new in this blog.
+
+We're extremely excited to announce the next milestone in this evolution, a public beta for dedicated clusters 🎉
+
+We also want to whole-heartedly thank our EAP customers for not only being early adopters, but providing amazing levels of feedback throughout the journey. Keep a look out, you'll all be receiving some premium Xata swag soon 💜
+
+## What is a dedicated cluster?
+
+Today, most Xata databases live in a shared cell architecture. Our multi-tenant cells include a Postgres Aurora cluster, an Elasticsearch cluster, high availability built in and all the APIs and services required for development. Within each cell, we split the system for each workspace to include about 2,000 active databases per cell before we have to spin up a new one. Unless you were an EAP customer, your Xata database is using this shared cell architecture.
+
+Our dedicated clusters are essentially dedicated cells. So you have all the same benefits of the shared cell architecture, with less limitations caused by shared resources, more control over your infrastructure, decoupled usage based pricing and the confidence that Xata can scale with your business.
+
+![Postgres branches allocated to shared and dedicated clusters](images/branches-clusters.png)
+
+I won't get into the weeds today, but if you're interested in learning more, [this blog](https://xata.io/blog/postgres-dedicated-clusters) and [our documentation](https://xata.io/docs/dedicated-cluster) are great resources.
+
+## Celebrating with a limited-time offer
```

Comment:
I wonder if the big-bold title here could literally be `## Get started with $1000 in credit` - where the $1000 would be super obvious + eye catching

Reply:
♥️ yess